### ISSUE TITLE
Default calendar and account access bad cycle

### ISSUE THREAD

**Issue by sergiopoverony** [aquaris-dev/DEPRECATED_android_device_bq_piccolo#105]
Default calendar ask account settings and find account on any run him.

**Comment by Kra1o5** [comment]
Hi @sergiopoverony ,

This is a upstream issue not device specific, if you use Google Calendar it works fine.

Greetings

**Comment by sergiopoverony** [comment]
Hi @Kra1o5 i know  about Google Calendar. One problem ))) i'm not use any google services

**Comment by Kra1o5** [comment]
In that case, you need to wait for a upstream fix.

**Comment by sergiopoverony** [comment]
Yes. I wait - for it i make this issues

**Comment by stucki** [comment]
No, you can just close this ticket because it's an upstream issue. It's device independent and is therefore fixed by other people who are not following this tracker.

Therefore, I suggest that you check this site and report the issue if it's not reported yet:
https://jira.cyanogenmod.org/

After this, add a link to the ticket here and close the issue. Thanks! :-)

**Comment by sergiopoverony** [comment]
HM/ Calendar is default CM app. Why not?

**Comment by stucki** [comment]
Because this project is only about porting CyanogenMod to one specific device. The other issue tracker is for general projects (like the default calendar) which are not device specific.
